src/Network/Network/VirtualRouterPeer/RemoveAzureRMVirtualRouterPeerCommand.cs

```diff
@@ -0,0 +1,124 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Management.Network;
+using System;
+using CNM = Microsoft.Azure.Commands.Network.Models;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Common.Strategies;
+using Microsoft.Azure.Management.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VirtualRouterPeer", SupportsShouldProcess = true, DefaultParameterSetName = VirtualRouterPeerParameterSetNames.ByVirtualRouterPeerName), OutputType(typeof(PSVirtualRouter))]
```

Comment:
As Jinlei said in https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/382#issuecomment-533966581, please implement `-PassThru` for `Remove-AzVirtualRouterPeer`

Reply:
As per the guidelines https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/cmdlet-best-practices.md, passthru is only needed when there is no output from cmdlet. RemoveAzVirtualRouterPeer return VirtualRouter object as output